### PR TITLE
Update Devise

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Update Devise to 3.5.5
* Update engines and install migrations

[Finishes: #113963033]

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -10,7 +9,17 @@ class ApplicationController < ActionController::Base
 
   helper ThinkFeelDoEngine::Engine.helpers
 
-  before_action :authenticate_participant!,
-                :instantiate_navigator,
+  before_action :set_raven_user_context,
                 :detect_browser
+
```
Comment (eschlange) [inline app/controllers/application_controller.rb]: Were these changes meant for this PR?

**Comment by eschlange** [comment]
@michaelwehrley Looks good though the addition of the experiences code isn't mentioned in the commit message. Were those changes meant for this pull request?

**Comment by michaelwehrley** [comment]
@eschlange They were part of update the gems with the install of migrations.  In the future, I think I will take that migration out of TFD Engine.

**Comment by eschlange** [comment]
Ah ok, merging.

**Pull request merged**